### DISCUSSION TITLE
Open multiple URLs at once

### DISCUSSION THREAD

**Discussion by DRYN07** [Flow-Launcher/Flow.Launcher#1898] [Q&A] [answered]
Is there any method (with or without plugins) to open multiple url-s at once with search queries?

For example search "dog" in multiple sites

https://site1?q=dog
https://site2?q=dog
https://site3?q=dog

**Comment by jjw24** [comment, marked as answer]
This might be what you are after as a workaround, see description's 'please note'
https://github.com/Flow-Launcher/Flow.Launcher/issues/438
